### ISSUE TITLE
Validation results show nan all the time

### ISSUE THREAD

**Issue by FairyFali** [cvignac/DiGress#51]
I encounter a strange result during validating. the result is 

Starting train epoch...
Epoch X: Val NLL nan -- Val Atom type KL nan --  Val Edge type KL: nan
Val loss: nan    Best val loss:  100000000.0000 

the NLL is always nan, why?

**Comment by FairyFali** [comment]
> Hello, this is not normal. What command are you running? Is it on a custom dataset?

I run the command as mentioned in the readme. it's on qm9 with discrete noise.

**Comment by cvignac** [comment]
I'm not sure where the exact issue came from (probably a different behavior of `mask_distributions` with recent python versions), but it's now fixed. You can use the latest commit.

**Comment by FairyFali** [comment]
> I'm not sure where the exact issue came from (probably a different behavior of `mask_distributions` with recent python versions), but it's now fixed. You can use the latest commit.

I think I figure it out. because you are using `.log` or `torch.log` in functions like `kl_prior`, and `compute_Lt` but not consider the zero to log(), so it will appear `nan`.
